### PR TITLE
Adds feature to input custom contentId to address issue #217.

### PR DESCRIPTION
**Pull request by nathanhere**
#217

### CODE REVIEW

File: src/Collapsible.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ class Collapsible extends Component {
 
     this.timeout = undefined;
 
-    this.contentId = `collapsible-content-${Date.now()}`;
+    this.contentId = props.contentElementId || `collapsible-content-${Date.now()}`;
```
Comment (glennflanagan) [inline src/Collapsible.js:12]: Could we add this prop to the types - and then we could actually use `collapsible-content-${Date.now()}` as the default prop for `contentElementId`

Comment (nathanhere) [reply src/Collapsible.js:12]: Done!

Comment (karltaylor) [reply src/Collapsible.js:12]: Would you mind adding to [index.d.ts](https://github.com/glennflanagan/react-collapsible/blob/develop/index.d.ts) to keep typescript happy too?

Comment (nathanhere) [reply src/Collapsible.js:12]: Done!

**Comment by Bat-Cave** [comment]
any idea when `contentElementId` will be merged into master? I'm trying to prevent duplicate id's for accessibility.

**Comment by karltaylor** [comment]
Hey @Bat-Cave, just released [2.9.0](https://github.com/glennflanagan/react-collapsible/releases/tag/2.9.0)

**Comment by Bat-Cave** [comment]
> Hey @Bat-Cave, just released [2.9.0](https://github.com/glennflanagan/react-collapsible/releases/tag/2.9.0)

Thank you so much!

**Pull request merged**